docs/scenarios.md

```diff
@@ -1,36 +1,34 @@
 
-To remove the #1 Broker from the cluster, we need to lower the number of brokers in the cluster from three to one. This will cause a state in which only one Broker is live, while we kill the brokers that handle traffic. {{< kafka-operator >}} supports removing specific brokers without disrupting traffic in the cluster.
+To remove the #1 Broker from the cluster, you need to lower the number of brokers in the cluster from three to one. This will cause a state in which only one Broker is live, while you kill the brokers that handle traffic. {{< kafka-operator >}} supports removing specific brokers without disrupting traffic in the cluster.
 
 ## Fine grained Broker config support
 
-Apache Kafka is a stateful application, where Brokers create/form a cluster with other Brokers. Every Broker is uniquely configurable (we support heterogenous environments, in which no nodes are the same, act the same or have the same specifications - from the infrastructure up through the Brokers' Envoy configuration). Kafka has lots of Broker configs, which can be used to fine tune specific brokers, and we did not want to limit these to ALL Brokers in a StatefulSet. We support unique Broker configs.
+Apache Kafka is a stateful application, where Brokers create/form a cluster with other Brokers. Every Broker is uniquely configurable ({{< kafka-operator >}} supports heterogenous environments, in which no nodes are the same, act the same or have the same specifications - from the infrastructure up through the Brokers' Envoy configuration). Kafka has lots of Broker configs, which can be used to fine tune specific brokers, and {{< kafka-operator >}} did not want to limit these to ALL Brokers in a StatefulSet. {{< kafka-operator >}} supports unique Broker configs.
 
-*In each of the three scenarios lister above, we decided to not use StatefulSet in our {{< kafka-operator >}}, relying, instead, on Pods, PVCs and ConfigMaps. We believe StatefulSet is very convenient starting point, as it handles roughly 80% of scenarios but introduces huge limitations when running Kafka on Kubernetes in production.*
+*In each of the three scenarios lister above, {{< kafka-operator >}} does not use StatefulSet, relying, instead, on Pods, PVCs and ConfigMaps. With {{< kafka-operator >}}, StatefulSet is very convenient starting point, as it handles roughly 80% of scenarios but introduces huge limitations when running Kafka on Kubernetes in production.*
```

Comment:
**Required**: We **MUST** clarify the wording here, Koperator doesn't use statefulset, other solutions using statefulset are fine to cover 80% of the cases, Koperator aims to cover 100%.

```suggestion
*In each of the three scenarios lister above, {{< kafka-operator >}} does not use StatefulSet, relying, instead, on Pods, PVCs and ConfigMaps. While using StatefulSet is a very convenient starting point, as it handles roughly 80% of scenarios, it also introduces huge limitations when running Kafka on Kubernetes in production.*
```